src/client/dispatcher.js

```diff
@@ -2,8 +2,9 @@ import {Dispatcher} from 'flux'
 
 const dispatcher = new Dispatcher
 
-export function register(callback: Function): string {
-  return dispatcher.register(callback)
+export function register(handlers:Object): string {
+  return dispatcher.register(({action, data}) =>
+    handlers[action] && handlers[action](data))
```

Comment:
Don't have to worry about break and also you can destructure the arguments in the lamba signature.